modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/AllSectionDisplayContext.java

```diff
@@ -83,7 +83,7 @@ public String getRootObjectEntryFolderExternalReferenceCode() {
 
 	@Override
 	protected String getCMSSectionFilterString() {
-		return "cmsContent eq true";
+		return "cmsSection eq 'contents' or cmsSection eq 'files'";
```

Comment:
To test, but in this way we are showing as well folders, while in the all section we have to show only content

Reply:
Yes, you are right. I'll fix it.